roles/ceph-common/tasks/installs/redhat_ceph_repository.yml

```diff
@@ -31,6 +31,39 @@
     backup: yes
   when: ceph_dev
 
+- name: fetch nfs-ganesha development repo file
+  uri:
+    url: https://shaman.ceph.com/api/repos/nfs-ganesha/next/latest/{{ ansible_distribution | lower }}/{{ ansible_distribution_major_version }}/flavors/{{ nfs_ganesha_flavor }}/repo
+    return_content: yes
+  register: nfs_ganesha_dev_yum_repo
+  when:
+    - nfs_ganesha_dev is defined
+    - nfs_ganesha_dev
+
+- name: add nfs-ganesha stable repository
+  yum_repository:
+    name: nfs_ganesha_stable
+    description: nfs-ganesha stable repo
+    gpgcheck: yes
+    state: present
+    gpgkey: "{{ ceph_stable_key }}"
+    baseurl: "{{ ceph_mirror }}/nfs-ganesha/rpm-{{ nfs_ganesha_stable_branch }}/{{ ceph_stable_release }}/$basearch"
+  when: 
+    - nfs_ganesha_stable is defined
+    - nfs_ganesha_stable
+
+- name: add nfs-ganesha development repository
+  copy:
+    content: "{{ nfs_ganesha_dev_yum_repo.content }}"
+    dest: /etc/yum.repos.d/nfs-ganesha-dev.repo
+    owner: root
+    group: root
+    backup: yes
+  when: 
+    - nfs_ganesha_dev is defined
+    - nfs_ganesha_dev_yum_repo is defined
+    - nfs_ganesha_stable
```

Comment:
Is this correct here? Don't you want either ``nfs_ganesha_stable`` or ``nfs_ganesha_dev``?